SIL.WritingSystems.Tests/SldrTests.cs

```diff
@@ -76,6 +76,13 @@ public void Setup()
 				Sldr.Initialize(true, Sldr.DefaultSldrCachePath);
 		}
 
+		[Test]
+		public static void CheckUserAgent()
+		{
+			Assert.That(Sldr.UserAgent, Does.Contain("SIL.WritingSystems Library")
+				.And.Contains(typeof(Sldr).Assembly.GetName().Version.ToString()));
```

Comment:
This will be the first place in libpalaso with `.And.` in a test assert; would it be better to split this into two asserts for consistent testing style?

Reply:
I don't think that's a problem. As long as the intention of the Assert is one fundamentally related check.